i18n/messages-en.xtb

```diff
@@ -813,16 +813,25 @@
   <translation id="5847167541694942228" key="MSG_PETSETDETAIL_PETSETINFO_9" desc="The message says that that many pods are pending (pet set details page).">{{ $ctrl.petSet.podInfo.pending}} pending</translation>
   <translation id="7332879554461510589" key="MSG_PETSETLIST_PETSETCARDLIST_0" desc="Column header \'Kind\' of a Pet Set List">Kind</translation>
   <translation id="2816069801101820166" key="MSG_PETSETLIST_PETSETCARDLIST_1" desc="Pet set list header: name.">Name</translation>
+  <translation id="3262303717958393654" key="MSG_PETSETLIST_PETSETCARDLIST_1" desc="Pet set list header: name.">Name</translation>
```

Comment:
Eek, lot of added duplications in this file. Not a problem with your code, but we might need to take a look at why this happens. Might be a problem in my i18n fix commit from PR #1386